### PR TITLE
Improve submodule bump workflow

### PR DESCRIPTION
**Pull request by lava**
* Require an open PR for any changes to the plugins submodule
* Remove some leftover debug code
* Rename the used secrets away from `TESTAPP`
* Fix a bug where the submodule would be bumped to an incorrect state.

### CODE REVIEW

**Review by dominiklohmann — APPROVED** [review]
The changes here all make sense to me.

File: .github/workflows/tenzir.yaml [hunk 1]
```diff
@@ -349,8 +349,24 @@ jobs:
...
+          app-id: ${{ vars.TENZIR_AUTOBUMPER_APP_ID }}
+          private-key: ${{ secrets.TENZIR_AUTOBUMPER_APP_PRIVATE_KEY }}
+          owner: ${{ github.repository_owner }}
+      - name: Require an open PR for the submodule bump
+        run: |
+          plugins_sha=$(git -C contrib/tenzir-plugins rev-parse HEAD)
+          plugins_main_sha=$(git -C contrib/tenzir-plugins rev-parse origin/main)
+
+          if [ "$plugins_sha" != "$plugins_main_sha" ]; then
+            pr_number=$(gh pr --repo tenzir/tenzir-plugins list --search $plugins_sha --json number -q .[0].number)
```
Comment (dominiklohmann) [inline .github/workflows/tenzir.yaml:367]: Can we also check here if all comments on the PR are resolved?

Comment (lava) [reply .github/workflows/tenzir.yaml:367]: As far as I can tell there's no easy way to do this using `gh` (or the github API for that matter).

I'm not even sure how to check that all tests have passed, there is a `mergeable` field but that only checks if there would be a merge conflict in the PR.

I think at least initially we'll have to rely on reviewers and authors to not merge until comments from both PRs are fully resolved.

Comment (dominiklohmann) [reply .github/workflows/tenzir.yaml:367]: Fine by me.

**Review by Dakostu — APPROVED** [review]

**Pull request merged**